core/chains/evm/logpoller/log_poller.go

```diff
@@ -147,6 +147,7 @@ func NewLogPoller(orm ORM, ec Client, lggr logger.Logger, pollPeriod time.Durati
 		backfillBatchSize:        backfillBatchSize,
 		rpcBatchSize:             rpcBatchSize,
 		keepFinalizedBlocksDepth: keepFinalizedBlocksDepth,
+		logPrunePageSize:         logsPrunePageSize,
```

Comment:
I've added this param as a backward compatibility layer. Products already using "delete all" queries will not be affected by this PR. Product has to explicitly specify that they need paging